src/core/qgsvectorlayerfeatureiterator.cpp

```diff
@@ -27,6 +27,7 @@
 #include "qgsmessagelog.h"
 #include "qgsexception.h"
 #include "qgsexpressioncontextutils.h"
+#include <QDebug>
```

Comment:
```suggestion
```

Reply:
remnant of local testing, I'll clean this next commit.